Integrations/src/main/java/io/deephaven/integrations/learn/Input.java

```diff
@@ -63,12 +64,13 @@ private Input(String[] colNames, Function<Object[], Object> gatherFunc) {
         for (int i = 0 ; i < colNames.length ; i++) {
             colSet[i] = table.getColumnSource(colNames[i]);
         }
+        table.select(colNames).getColumnSources().toArray(ColumnSource.ZERO_LENGTH_COLUMN_SOURCE_ARRAY);
```

Comment:
```suggestion
```
I suspect this got committed by accident.  It does a ton of unnecessary computation.  `select` is creating a whole new in-memory table.